app/src/main/java/ch/epfl/sdp/blindwar/game/util/DisplayableItemAdapter.kt

```diff
@@ -316,6 +386,44 @@ class DisplayableItemAdapter(
         const val DURATION_FAST = 20000L
         const val DURATION_DEFAULT = 30000L
     }
+

```

Comment:
Try to move the share logic in a separate file to reduce the number of lines in this file